### PR TITLE
NEPT-2414: Integrate EC Europa theme version 0.1.x in NextEuropa

### PR DESCRIPTION
**Pull request by aritomelo**
### Description
Integrate EC Europa theme version 0.1.x in NextEuropa

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
     <property file="build.properties.local" />
     <property file="build.properties" />
+    <property file="build.properties.docker" />
```
Comment (drupol) [inline build.xml]: This is out of scope.

**Review by drupol — CHANGES_REQUESTED** [review]

**Review by Fefaine — CHANGES_REQUESTED** [review]
We now use the "semantic versioning" (or simili ;) ) so we base all our commits for next release to release-2.5.137 branch.
That's the branch that will be QAd and merged to 2.5 once the QA is validated.
Can you therefore rebase your PR to that branch? Thanks !

**Comment by drupol** [comment]
Thanks @Fefaine , it's done.

**Review by Fefaine — APPROVED** [review]

**Review by drupol — APPROVED** [review]
@aritomelo great job ! Approving!

**Pull request merged**